File: Data Structures & Algorithms/concatenation-of-array/submission-3.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int n = nums.length;
        int[] arr = new int[2 * n];
        int idx = 0;
        for(int i = 0; i < n; i++){
            arr[i] = arr[i + n] = nums[i];
        }
        return arr;
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-3.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        seen = {}
        for num in nums:
            seen[num] = 1 + seen.get(num, 0)
        arr = []
        for i , val in seen.items():
            arr.append([val,i])
        arr.sort()
        res = []
        while len(res) < k:
            res.append(arr.pop()[1])
        return res
        
     